### DISCUSSION TITLE
Push logs of different namespaces to different kafka clusters

### DISCUSSION THREAD

**Discussion by ShahroZafar** [vectordotdev/vector#20566] [Q&A] [answered]
We have a use case where we want to send logs of different namespaces to different kafka clusters. Is it possible to dynamically set the inputs or broker endpoints to achieve this?

**Comment by jszwedko** [comment, marked as answer]
Unfortunately not, you would need seperate `kafka` sinks to be defined and to use a `route` transform to route between them.

**Reply by ShahroZafar** [reply]
Thanks. 
We will give it a try.
